src/randoop/main/Main.java

```diff
@@ -87,9 +89,26 @@ public void nonStaticMain(String[] args) {
       System.out.println("`help " + handler.fcommand + "' as arguments.");
       System.exit(1);
 
+    } catch (SequenceExceptionError e) {
+      
+      System.out.println();
+      String msg;
```

Comment:
I would rather have one statement that declares and initializes the variable, rather than two.
Is the variable even necessary here?  Writing System.out.println on each line would actually be shorter than the current construction.